gunicorn.conf.py

```diff
@@ -0,0 +1,71 @@
+import uvicorn
```

Comment:
@DmytroLyt you created your branch (EU-112) not from release branch (eu-release-0.1.0), but from EU-84. It's wrong. Please, re-create your branch from eu-release-0.1.0, move needed changes in in new branch and re-create PR